rules/no-inheritance.js

```diff
@@ -15,26 +15,69 @@ module.exports = {
             description: 'Discourage use of inheritance'
         },
 
-        schema: []
+        schema: [{
+            type: 'object',
+            properties: {
+                'no-interface': { type: 'boolean' }
+            }
```

Comment:
add `additionalProperties: false` after `properties` object. We want to keep schema as strict and narrowed as we can.
```
schema: [{
  type: 'object',
  properties: {...},
  additionalProperties: false
}]
```